### DISCUSSION TITLE
Firefox rendering error

### DISCUSSION THREAD

**Discussion by zhenzhenChange** [community/community#64149] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

```text
System:
    OS: macOS 13.5
    CPU: (10) arm64 Apple M2 Pro
    Memory: 663.72 MB / 16.00 GB

Firefox 116.0.3 (64)
```

There is a probability of overlap when typing/pasting text in the input box.

Related topics: https://support.mozilla.org/en-US/questions/1331767

<img width="982" alt="image" src="https://github.com/community/community/assets/51586637/5aa3e2c7-10f7-49bc-ac6e-77168a8bc5f0">

**Comment by queenofcorgis** [comment]
Thank you for flagging this our way. I've added the appropriate label. Any additional details you can share are helpful - like is this in all input boxes across the site or is it specific to a feature(s)?

**Comment by zhenzhenChange** [comment]
I've recently discovered that it's actually not just the `GitHub` site that does this, but other sites' input boxes as well, which should have nothing to do with the site, and appears to be a `Firefox` bug.

But I don't know how to submit a bug to `Firefox`, or maybe someone has submitted it but hasn't fixed it yet.

e.g. - http://demo.showdownjs.com/

<img width="540" alt="图片" src="https://github.com/community/community/assets/51586637/bd8f3673-fa6b-46f6-a0d7-7f158fbf621c">
